src/modules/asab-config/index.js

```diff
@@ -25,11 +25,10 @@ export default class ConfigModule extends Module {
 			component: ConfigContainer,
 		});
 
-		// TODO: add routes for types and configs
 
 		this.Navigation.addItem({
 			name: "Configuration",
-			url: "/config/*/*",
+			url: "/config/type/name",
```

Comment:
Better 'empty state' variables are recommended here ... such a `/config/$/$` so that you prevent a collision with reasonably named files.